airflow/ti_deps/ti_deps.py

```diff
@@ -0,0 +1,479 @@
+from abc import ABCMeta, abstractmethod
```

Comment:
I kind of like @abstractmethod since it will warn you about problems once modules are loaded (e.g. when a webserver is started), rather than when a specific function is called. The faster you fail the better, e.g. compilation failures are better than runtime failures). Not a big deal to me though if you feel strongly about it.